sonarr/series.go

```diff
@@ -137,3 +237,27 @@ func (s *Sonarr) Lookup(term string) ([]*Series, error) {
 func (s *Sonarr) LookupContext(ctx context.Context, term string) ([]*Series, error) {
 	return s.GetSeriesLookupContext(ctx, term, 0)
 }
+
+// DeleteSeries removes a single Series.
+// deleteFiles and importExclude flags defines the deleteFiles and addImportListExclusion query paramenters respectively.
+func (s *Sonarr) DeleteSeries(seriesID int, deleteFiles bool, importExclude bool) error {
+	return s.DeleteSeriesContext(context.Background(), seriesID, deleteFiles, importExclude)
+}
+
+func (s *Sonarr) DeleteSeriesContext(ctx context.Context, seriesID int, deleteFiles bool, importExclude bool) error {
+	params := make(url.Values)
+	params.Add("deleteFiles", strconv.FormatBool(deleteFiles))
+	params.Add("addImportListExclusion", strconv.FormatBool(importExclude))
+
+	uri := path.Join(bpSeries, strconv.Itoa(seriesID))
+	if _, err := s.Delete(ctx, uri, params); err != nil {
+		return fmt.Errorf("api.Delete(%s): %w", uri, err)
+	}
+
+	return nil
+}
+
+// Default behaviour is to set deleteFiles to true and addImportListExclusion to false.
```

Comment:
Comments should start with `// DeleteSeriesDefault  ...`

Reply:
correct, I'm going to change this before merging